DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/vlan_suite/test_all_supported_modes.py

```diff
@@ -0,0 +1,186 @@
+import pytest
+
+from dent_os_testbed.lib.bridge.bridge_vlan import BridgeVlan
+from dent_os_testbed.utils.test_utils.tgen_utils import tgen_utils_get_dent_devices_with_tgen
+
+pytestmark = pytest.mark.suite_functional_vlan
+
+packet_vids = ['x', 1, 2, 3, 4]  # VLAN tag number contained in the transmitted packet
+vlans = {
+    2: {
+        "ports": [1, 2],
+        "pvid": False,
+        "untagged": False
+    },
+    3: {
+        "ports": [1],
+        "pvid": False,
+        "untagged": True
+    },
+    "3": {
+        "ports": [3],
+        "pvid": True,
+        "untagged": True
+    },
+    4: {
+        "ports": [1, 4],
+        "pvid": True,
+        "untagged": True
+    }
+}
+
+
+@pytest.mark.asyncio
```

Comment:
please move this (and all other decorators) to L6:
```python
pytestmark = [
    pytest.mark.suite_functional_vlan,
    pytest.mark.asyncio,
]
```
This will mark all of your test in this file as 'async' and 'suite_functional_vlan'.